apps/artist/routes.coffee

```diff
@@ -20,13 +20,16 @@ query = """
 module.exports.index = (req, res, next) ->
   artist = new Artist id: req.params.id
   artist.fetch
+    error: res.backboneError
     cache: true
     success: ->
       artist.maybeFetchAndSetFeaturedBio =>
-        res.locals.sd.ARTIST = artist.toJSON()
-        showAuctionLink = artist.get('display_auction_link')
-        res.render 'page', artist: artist, sort: req.query?.sort, showAuctionLink: showAuctionLink
-    error: res.backboneError
+        artist.fetchArtworks
```

Comment:
Because of all the callbacks, this might be a good candidate for using a promise. It might be easier to stub as well. Here is an [example](https://github.com/artsy/microgravity/blob/master/apps/fair_info/routes.coffee#L17) of a pattern you might want to follow and here are the [corresponding test](https://github.com/artsy/microgravity/blob/master/apps/fair_info/test/routes.coffee#L23-L36).

Let me know if you want to pair on this.